src/Microsoft.DotNet.Interactive/Parsing/DirectiveNode.cs

```diff
@@ -343,15 +343,20 @@ public async Task<DirectiveBindingResult<string>> TryGetJsonAsync(DirectiveBindi
 
         return DirectiveBindingResult<string>.Success(json);
 
-        void WriteProperty(string propertyName, string value)
+        void WriteProperty(string propertyName, string value, bool containsJson)
         {
-            if (value.Length > 0 && value[0] is '{' or '"' or '[')
+            if (containsJson)
             {
                 writer.WritePropertyName(propertyName);
                 writer.WriteRawValue(value);
             }
             else
             {
+                if (value.Length > 0 && value[0] is '"')
+                {
+                    value = value.Trim('"');
```

Comment:
The value cannot contain leading and trailing whitespace, can it?

Reply:
No, because we're using the `Text` property, not the `FullText` property, so trivia has already ben removed.